ctx.go

```diff
@@ -1054,6 +1054,40 @@ func (c *Ctx) Redirect(location string, status ...int) error {
 	return nil
 }
 
+// Redirect to the URL derived from the specified path, with specified status.
+// If status is not specified, status defaults to 302 Found.
+func (c *Ctx) Redirect(location string, status ...int) error {
+	return c.redirect(location, status...)
+}
+
+// RedirectToRoute to the Route registered in the app with appropriate parameters
+// If status is not specified, status defaults to 302 Found.
+func (c *Ctx) RedirectToRoute(routeName string, params Map, status ...int) error {
+	route := c.App().GetRoute(routeName)
+	location := ""
+	for _, segment := range route.routeParser.segs {
+		location = fmt.Sprintf("%s%s", location, segment.Const)
+		if segment.IsParam {
+			if val, ok := params[segment.ParamName]; ok {
+				location = fmt.Sprintf("%s%s", location, val)
```

Comment:
please add a benchmark method for this

would like to have the possibility to optimize this later on

Reply:
Done